src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -55,6 +56,22 @@ describe(RundownTimelineService.name, () => {
 
       await expect(() => testee.deleteRundown(rundown.id)).rejects.toThrow(ActiveRundownException)
     })
+
+    it('throws an exception, when trying to active a rundown when there is another already activated rundown', async () => {
+      const mockRundownRepository: RundownRepository = mock<RundownRepository>()
+
+      const activeBasicRundown: Rundown = EntityMockFactory.createRundown({ isRundownActive: true })
+
+      const basicRundownArray: Rundown[] = [activeBasicRundown];
+
+      const rundownToActivate: Rundown = EntityMockFactory.createRundown({isRundownActive: false})
```

Comment:
Since you are making two different Rundowns it would be a good idea to also provide an id.
The current implementation of `EntityMockFactory.createRundown()` defaults the `rundownId` to be "rundownId" if no other id is provided. So right now you have made two different Rundowns with the same id.

All you would to do is add the id like this:
`EntityMockFactory.createRundown({ id: 'inactiveRundown', isRundownActive: false })`

Reply:
I have made the requested changes.